### ISSUE TITLE
Weird issue with music wait=False

### ISSUE THREAD

**Issue by gbaman** [bbcmicrobit/micropython#218]
Was fiddling around with doing piano stuff, where the note doesn't stop till something else stops it (holding down the key on the piano then let go).
I had this code below.
```python
from microbit import *
import music
print("Starting")

sleep(1000)

music.play('c4:4', wait=False)

while True:
    print("Test")
    sleep(1000)
```
When I ran it on the Microbit using Mu, got a bit of a surprising result... Is only an issue when I set ```wait=False```, which is required to do my idea and use ```.stop()``` later on.
![screen shot 2016-02-23 at 00 40 28](https://cloud.githubusercontent.com/assets/833193/13302090/fabfb72c-db3f-11e5-9fb3-5c94be18a205.png)

**Comment by dpgeorge** [comment]
Yep, that is weird!  Likely it's something to do with trying to convert an object to a string to play the note, but instead of a string there is a boolean.  The strange part is, where does the boolean come from?  I'll look into it.

**Comment by dpgeorge** [comment]
Work around is to put the single note in a list:
```
music.play(['c4:4'], wait=False)
```

**Comment by ntoll** [comment]
Fixed by #219
